package.json

```diff
@@ -2,14 +2,15 @@
   "name": "agenda",
   "version": "4.0.0",
   "description": "Light weight job scheduler for Node.js",
-  "main": "index.js",
+  "main": "dist/index.js",
   "files": [
```

Comment:
we could add 
  "types": "dist/index.d.ts",
here.. (not sure if we have types with the curernt build config?, but if we have, adding this would give the user a lot of beneftis when using this library :))

Reply:
Agree, after fixing the release, we should add types-file with the xports.

Reply:
Yep yep, a good follow up task. 👍